### ISSUE TITLE
Panic: Failed to set CRTC info

### ISSUE THREAD

**Issue by sunweaver** [AyatanaIndicators/ayatana-indicator-a11y#21]
I have tinkered with my VM setup (Spice -> VGA/VNC display, not sure if related) and after that, ayatana-indicator-a11y crashes on startup with this error message:

```

process:<n>): ayatana-indicator-a11y-ERROR **: <time>: Panic: Failed to set CRTC info
```

The scaling is set to 1.5 (org.ayatana.indicator.a11y scale).

**Comment by sunweaver** [comment]
When setting the scale back to 1, the a11y indicator comes up again, and it also crashes when using the scaling slider and settings it on some value  other than 100%.
